webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue

```diff
@@ -147,10 +146,10 @@ export default {
       }
       CKEDITOR.dtd.$removeEmpty['i'] = false;
 
-      let extraPlugins = 'simpleLink,suggester,widget,emoji';
+      let extraPlugins = 'simpleLink,suggester,widget,emoji,formatOption';
       let removePlugins = 'image,maximize,resize';
       const toolbar = [
-        ['Bold', 'Italic', 'BulletedList', 'NumberedList', 'Blockquote'],
+        ['formatOption','Bold', 'Italic', 'BulletedList', 'NumberedList', 'Blockquote'],
```

Comment:
the new format option should be added only when newEditorToolbarEnabled

Reply:
Done!